cypress/integration/layerErrorIcon.spec.js

```diff
@@ -0,0 +1,17 @@
+describe('Broken icon for layer errors', function() {
```

Comment:
@sagarpreet-chadha, is there a way you are aware of where I can make a stub send a request error in its response. I want to try to mimic xhr errors but I can't seem to figure out how.